### ISSUE TITLE
-bug- Fixed thank you for creating with WordPress bottom footer message

### ISSUE THREAD

**Issue by paaljoachim** [CalderaWP/Caldera-Forms#1671]
Hey

It seems the thank you for creating with WordPress and the version is fixed on a submission page. It looks like this here:

Look for the text: "Takk for at du skaper med WordPress" and to the right version 4.8.

![screen shot 2017-06-14 at 22 41 54](https://user-images.githubusercontent.com/5323259/27153725-d984e6b0-5152-11e7-9302-47d6348ea551.png)

It is only on the submission page.

On the all forms page it looks like this. (As it should)

![screen shot 2017-06-14 at 22 42 09](https://user-images.githubusercontent.com/5323259/27153737-e749aa42-5152-11e7-8e72-c8af27ebd20b.png)

**Comment by Shelob9** [comment]
@paaljoachim Can you please test on a fresh install with just the default theme? I don't see this, and suspect it is a theme or plugin conflict. If you can reproduce this with just Caldera Forms and Twenty Seventeen, please open a new issue using the issue template.

**Comment by paaljoachim** [comment]
I tested this on two local dev sites. One older and one new fresh site. Both use Twenty Seventeen and very few plugins.

The older shows this behavior (even after turning off all plugins except Caldera Forms).
The new site does not show this behavior.

**Comment by paaljoachim** [comment]
Another comment. This is what it looks like on an older site.
(I set the zoom at 67% to show multiple issues. The Thank you for creating with WordPress footer text and the Json code that is showing up.

![screen shot 2017-06-16 at 23 33 50](https://user-images.githubusercontent.com/5323259/27249837-72068516-52ec-11e7-982c-d8002bdadfd1.png)

Btw One also has to scroll to the bottom to scroll sideways to get the fields on the right side to show up, but since I zoomed out all the fields show up.
